pycqed/measurement/calibration/single_qubit_gates.py

```diff
@@ -233,48 +321,255 @@ def __init__(self, task_list, sweep_points=None, **kw):
             self.preprocessed_task_list, self.sweep_block, **kw)
 
     def resolve_lo_sweep_points(self, freq_sp_suffix='freq', **kw):
```

Comment:
Is this the best name for this method? Many more things are done than create lo sweep points. Could we split this huge method that does many things into smaller methods (one for creating lo sweep points, one for lo offsets, one for cal points pulse modifs, one for drive_amp_adaptation, one for ro_freq_adaptation, and one for ro_flux_amp_adaptation for instance)? Then if each have docstrings i think the code would be much more readable. But I do understand this requires more work.

Reply:
It would also require retesting everything, so it sounds like a quite bad benefit/cost ratio and will prevent us from moving forward. You can add a FIXME or an issue that this should be done in the future, if you want.

I could rename it to resolve_freq_sweep_points since everything that is configured here is swept together with the frequency sweep points. Would this help?

Reply:
Okay makes sense to not do it now. 
Yes, I would find the new name more meaningful.